wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,11 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'fill-200x200|height-150')
```

Comment:
If the filter spec contains spaces, could we change them for a different character when generating the image filename? (I think spaces in filenames looks messy and they have to be urlencoded)